EventsExpress/ClientApp/src/actions/register/register-complete-action.js

```diff
@@ -24,6 +25,10 @@ export default function registerComplete(data, { shouldSaveMoreInfo }) {
         const jsonRes = await profileResponse.json();
         localStorage.setItem(jwtStorageKey, jsonRes.token);
 
+        if (data.image !== undefined) {
```

Comment:
shouldn't we exclude the image from the previous call? something like
```
const { image, ...profileData } = data;
const body = {
    ...profileData,
    birthday: moment.utc(profileData.birthday).local().format('YYYY-MM-DD[T00:00:00]'),
};
```

Reply:
Good idea